doc/environment_variables.md

```diff
@@ -89,6 +89,7 @@ some configuration as environment variables that can be set.
   - pending_tags - traces still-in-progress tags on completion queues
   - polling - traces the selected polling engine
   - polling_api - traces the api calls to polling engine
+  - subchannel
```

Comment:
Please append a description about what kinds of events will this tracer track.

From the references to this tracer, I see the following are tracked:
1. `ConnectedSubchannel` refcounting
2. subchannel connectivity change
3. subchannel refcounting